tests/test_fmi3.py

```diff
@@ -790,6 +790,50 @@ def test_generation_tool(self):
         assert "Reference FMUs" in fmu.get_generation_tool()
 
 
+class Test_FMI3Alias:
+    """Various tests surrounding aliases in FMI3."""
+    @classmethod
+    def setup_class(cls):
+        path = str(this_dir / "files" / "FMUs" / "XML" / "ME3.0" / "alias")
+        cls.fmu = FMUModelME3(path, allow_unzipped_fmu = True, _connect_dll = False)
+        
+    def test_get_model_variables_with_alias(self):
+        """Test get_model_variables including aliases."""
+        model_vars = self.fmu.get_model_variables(include_alias = True)
+        assert "v5" in model_vars
+        assert "v5_a1" in model_vars
+        assert "v5_a2" in model_vars
+
+        assert model_vars["v5"].description == "v5_desc"
+        assert model_vars["v5_a1"].description == ""
+        assert model_vars["v5_a2"].description == "v5_a2_desc"
```

Comment:
Maybe also just add check that the class attribute `.alias` is True or False as expected.

Reply:
Added.